source/2-our-operating-process.md

```diff
@@ -19,11 +19,11 @@ Example events include:
 
 If you have files, documents, or images, you can drop them into the Slack chat room for group sharing. This practice eliminates cumbersome, back-and-forth processes such as sending multiple emails to several people or adding individual accounts to Dropbox shared folders.
 
-In addition, we document our progress via GitHub (our preferred source control system) and Waffle (our preferred project management system). We put multiple processes in place so our clients never need to ask, “How is everything going?”. We try to use chat as much as possible so we can communicate frequently and informally.
+In addition, we document our progress via GitHub (our preferred source control system) and Waffle (our preferred project management system). We put multiple processes in place so our clients never need to ask, "How is everything going?". We try to use chat as much as possible so we can communicate frequently and informally.
```

Comment:
Also something that might be helpful here is to add that we ensure that a staging environment is setup so that the progress of the app can be viewed at any time?